XML Comments - Badly formed XML comment

When the Stride Docs is generated we get many warnings which cause certain parts of the Stride code being excluded from the Stride Docs.

**Describe the bug**
docfx warning example: Invalid triple slash comment is ignored:  Badly formed XML comment ignored for member "P:Stride.Core.Serialization.IUrlReference.Url"

**Expected behavior**
The triple slash must be present

**Screenshots**
<img width="381" alt="image" src="https://github.com/stride3d/stride/assets/4528464/f4afd537-1933-49d7-ae20-b3fb1493a98b">

**Additional context**
The Stride code XML comments are used to improve Stride Docs so we have to make sure all XML comments are valid, including spending time on creating comments.

I am going to list all warnings, this issue should focus only on Badly formed XML comment, search below for `invalid triple`

```
Loading project D:/Projects/GitHub/stride/sources/assets/Stride.Core.Assets/Stride.Core.Assets.csproj
  Determining projects to restore...
  All projects are up-to-date for restore.
D:\Projects\GitHub\stride\sources\engine\Stride.Audio\CompressedSoundSource.cs(20,24): warning CS0618: 'UnmanagedArray<short>' is obsolete: 'Obtain Memory<T> using GC.Allocate*Array or a Stride-specific allocator mechanism.'
D:\Projects\GitHub\stride\sources\engine\Stride.Rendering\Properties\AssemblyInfo.cs(5,49): warning CS0436: The type 'PublicKeys' in 'D:\Projects\GitHub\stride\sources\shared\SharedAssemblyInfo.cs' conflicts with the imported type 'PublicKeys' in 'Stride.Graphics, Version=4.1.0.1, Culture=neutral, PublicKeyToken=null'. Using the type defined in 'D:\Projects\GitHub\stride\sources\shared\SharedAssemblyInfo.cs'.
D:\Projects\GitHub\stride\sources\engine\Stride.Rendering\Properties\AssemblyInfo.cs(6,49): warning CS0436: The type 'PublicKeys' in 'D:\Projects\GitHub\stride\sources\shared\SharedAssemblyInfo.cs' conflicts with the imported type 'PublicKeys' in 'Stride.Graphics, Version=4.1.0.1, Culture=neutral, PublicKeyToken=null'. Using the type defined in 'D:\Projects\GitHub\stride\sources\shared\SharedAssemblyInfo.cs'.
D:\Projects\GitHub\stride\sources\engine\Stride.Rendering\Properties\AssemblyInfo.cs(7,62): warning CS0436: The type 'PublicKeys' in 'D:\Projects\GitHub\stride\sources\shared\SharedAssemblyInfo.cs' conflicts with the imported type 'PublicKeys' in 'Stride.Graphics, Version=4.1.0.1, Culture=neutral, PublicKeyToken=null'. Using the type defined in 'D:\Projects\GitHub\stride\sources\shared\SharedAssemblyInfo.cs'.
D:\Projects\GitHub\stride\sources\assets\Stride.Core.Assets\IAssetComposite.cs(17,21): warning CS0618: 'AssetPart' is obsolete: 'This struct might be removed soon'
D:\Projects\GitHub\stride\sources\assets\Stride.Core.Assets\PackageSession.cs(1578,22): warning CS0659: 'PackageSession.PendingPackageUpgrade' overrides Object.Equals(object o) but does not override Object.GetHashCode()
D:\Projects\GitHub\stride\sources\engine\Stride.Physics\IHeightStickArraySource.cs(26,24): warning CS0618: 'UnmanagedArray<T>' is obsolete: 'Obtain Memory<T> using GC.Allocate*Array or a Stride-specific allocator mechanism.'
D:\Projects\GitHub\stride\sources\engine\Stride.Physics\HeightStickArraySourceFromHeightmap.cs(34,31): warning CS0618: 'UnmanagedArray<T>' is obsolete: 'Obtain Memory<T> using GC.Allocate*Array or a Stride-specific allocator mechanism.'
D:\Projects\GitHub\stride\sources\engine\Stride.Physics\Shapes\HeightfieldColliderShape.cs(129,16): warning CS0618: 'UnmanagedArray<short>' is obsolete: 'Obtain Memory<T> using GC.Allocate*Array or a Stride-specific allocator mechanism.'
D:\Projects\GitHub\stride\sources\engine\Stride.Physics\ByteHeightStickArraySource.cs(45,31): warning CS0618: 'UnmanagedArray<T>' is obsolete: 'Obtain Memory<T> using GC.Allocate*Array or a Stride-specific allocator mechanism.'
D:\Projects\GitHub\stride\sources\engine\Stride.Physics\Shapes\HeightfieldColliderShape.cs(131,16): warning CS0618: 'UnmanagedArray<byte>' is obsolete: 'Obtain Memory<T> using GC.Allocate*Array or a Stride-specific allocator mechanism.'
D:\Projects\GitHub\stride\sources\engine\Stride.Physics\Shapes\HeightfieldColliderShape.cs(133,16): warning CS0618: 'UnmanagedArray<float>' is obsolete: 'Obtain Memory<T> using GC.Allocate*Array or a Stride-specific allocator mechanism.'
D:\Projects\GitHub\stride\sources\engine\Stride.Physics\UnmanagedArrayExtensions.cs(18,41): warning CS0618: 'UnmanagedArray<T>' is obsolete: 'Obtain Memory<T> using GC.Allocate*Array or a Stride-specific allocator mechanism.'
D:\Projects\GitHub\stride\sources\engine\Stride.Physics\ShortHeightStickArraySource.cs(45,31): warning CS0618: 'UnmanagedArray<T>' is obsolete: 'Obtain Memory<T> using GC.Allocate*Array or a Stride-specific allocator mechanism.'
D:\Projects\GitHub\stride\sources\engine\Stride.Physics\Shapes\HeightfieldColliderShape.cs(42,86): warning CS0618: 'UnmanagedArray<short>' is obsolete: 'Obtain Memory<T> using GC.Allocate*Array or a Stride-specific allocator mechanism.'
D:\Projects\GitHub\stride\sources\engine\Stride.Physics\Shapes\HeightfieldColliderShape.cs(47,86): warning CS0618: 'UnmanagedArray<byte>' is obsolete: 'Obtain Memory<T> using GC.Allocate*Array or a Stride-specific allocator mechanism.'
D:\Projects\GitHub\stride\sources\engine\Stride.Physics\Shapes\HeightfieldColliderShape.cs(52,86): warning CS0618: 'UnmanagedArray<float>' is obsolete: 'Obtain Memory<T> using GC.Allocate*Array or a Stride-specific allocator mechanism.'
D:\Projects\GitHub\stride\sources\engine\Stride.Physics\FloatHeightStickArraySource.cs(35,31): warning CS0618: 'UnmanagedArray<T>' is obsolete: 'Obtain Memory<T> using GC.Allocate*Array or a Stride-specific allocator mechanism.'
D:\Projects\GitHub\stride\sources\engine\Stride.Physics\Data\HeightfieldColliderShapeDesc.cs(81,24): warning CS0618: 'UnmanagedArray<T>' is obsolete: 'Obtain Memory<T> using GC.Allocate*Array or a Stride-specific allocator mechanism.'
Processing Stride.Core
Invalid cref value "!:LoadLibrary" found in triple-slash-comments for UnLoad defined in ../../stride/sources/core/Stride.Core/Native/NativeLibraryHelper.cs Line 164, ignored.
Invalid cref value "!:LoadLibrary" found in triple-slash-comments for UnLoadAll defined in ../../stride/sources/core/Stride.Core/Native/NativeLibraryHelper.cs Line 182, ignored.
Invalid cref value "!:T" found in triple-slash-comments for Equals defined in ../../stride/sources/core/Stride.Core/ReferenceEqualityComparer.cs Line 26, ignored.
Invalid cref value "!:T" found in triple-slash-comments for Empty defined in ../../stride/sources/core/Stride.Core/Collections/ArrayHelper.cs Line 16, ignored.
Invalid cref value "!:T" found in triple-slash-comments for this[] defined in ../../stride/sources/core/Stride.Core/Collections/FastCollection.cs Line 97, ignored.
Invalid cref value "!:T" found in triple-slash-comments for this[] defined in ../../stride/sources/core/Stride.Core/Collections/FastCollection.cs Line 97, ignored.
Invalid cref value "!:" found in triple-slash-comments for OldItem defined in ../../stride/sources/core/Stride.Core/Collections/FastTrackingCollectionChangedEventArgs.cs Line 42, ignored.
Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "M:Stride.Core.Collections.MultiValueSortedDictionary`2.CopyTo(System.Collections.Generic.KeyValuePair{`0,`1}[],System.Int32)" -->

Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "P:Stride.Core.Collections.MultiValueSortedDictionary`2.Enumerator.Current" -->

Invalid cref value "!:T" found in triple-slash-comments for this[] defined in ../../stride/sources/core/Stride.Core/Collections/OrderedCollection.cs Line 81, ignored.
Invalid cref value "!:" found in triple-slash-comments for OldItem defined in ../../stride/sources/core/Stride.Core/Collections/TrackingCollectionChangedEventArgs.cs Line 42, ignored.
Invalid cref value "!:Type.Name" found in triple-slash-comments for TypeName defined in ../../stride/sources/core/Stride.Core/Diagnostics/ExceptionInfo.cs Line 65, ignored.
Invalid cref value "!:Begin()" found in triple-slash-comments for Profiler defined in ../../stride/sources/core/Stride.Core/Diagnostics/Profiler.cs Line 56, ignored.
Invalid cref value "!:Profiler.Begin()" found in triple-slash-comments for ProfilingState defined in ../../stride/sources/core/Stride.Core/Diagnostics/ProfilingState.cs Line 15, ignored.
Invalid cref value "!:stream" found in triple-slash-comments for ToNativeStream defined in ../../stride/sources/core/Stride.Core/IO/NativeStreamExtensions.cs Line 23, ignored.
Invalid cref value "!:byte[]" found in triple-slash-comments for op_Explicit defined in ../../stride/sources/core/Stride.Core/Storage/ObjectId.cs Line 101, ignored.
Processing Stride.Core.IO
Invalid cref value "!:target" found in triple-slash-comments for CreateRelativePath defined in ../../stride/sources/core/Stride.Core.IO/VirtualFileSystem.cs Line 476, ignored.
Invalid cref value "!:sourcePath" found in triple-slash-comments for CreateRelativePath defined in ../../stride/sources/core/Stride.Core.IO/VirtualFileSystem.cs Line 476, ignored.
Processing Stride.Core.Mathematics
Processing Stride.Core.MicroThreading
Invalid cref value "!:Channel" found in triple-slash-comments for Channel defined in ../../stride/sources/core/Stride.Core.MicroThreading/Channel.cs Line 17, ignored.
Invalid cref value "!:Channel.Preference" found in triple-slash-comments for Channel defined in ../../stride/sources/core/Stride.Core.MicroThreading/Channel.cs Line 17, ignored.
Invalid cref value "!:RaiseException" found in triple-slash-comments for Exception defined in ../../stride/sources/core/Stride.Core.MicroThreading/MicroThread.cs Line 96, ignored.
Processing Stride.Core.Serialization
Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "P:Stride.Core.Serialization.IUrlReference.Url" -->

Invalid cref value "!:T" found in triple-slash-comments for UrlReference defined in ../../stride/sources/core/Stride.Core.Serialization/Serialization/UrlReference.cs Line 35, ignored.
Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "P:Stride.Core.Serialization.UrlReferenceBase.Url" -->

Invalid cref value "!:SaveStream" found in triple-slash-comments for CreateStream defined in ../../stride/sources/core/Stride.Core.Serialization/Storage/BundleOdbBackend.cs Line 778, ignored.
Invalid cref value "!:SaveStream" found in triple-slash-comments for CreateStream defined in ../../stride/sources/core/Stride.Core.Serialization/Storage/FileOdbBackend.cs Line 142, ignored.
Invalid cref value "!:SaveStream" found in triple-slash-comments for CreateStream defined in ../../stride/sources/core/Stride.Core.Serialization/Storage/IOdbBackend.cs Line 58, ignored.
Invalid cref value "!:SaveStream" found in triple-slash-comments for CreateStream defined in ../../stride/sources/core/Stride.Core.Serialization/Storage/ObjectDatabase.cs Line 323, ignored.
Invalid cref value "!:SaveStream" found in triple-slash-comments for CreateStream defined in ../../stride/sources/core/Stride.Core.Serialization/Storage/ObjectDatabase.cs Line 323, ignored.
Processing Stride
Invalid cref value "!:ParameterKeyProcessor" found in triple-slash-comments for AssemblyEffectKeysAttribute defined in ../../stride/sources/engine/Stride/Effects/AssemblyEffectKeysAttribute.cs Line 10, ignored.
Invalid cref value "!:EffectKeysAttribute" found in triple-slash-comments for AssemblyEffectKeysAttribute defined in ../../stride/sources/engine/Stride/Effects/AssemblyEffectKeysAttribute.cs Line 10, ignored.
Invalid cref value "!:IDepthStencilState" found in triple-slash-comments for CompareFunction defined in ../../stride/sources/engine/Stride/Graphics/CompareFunction.cs Line 14, ignored.
Invalid cref value "!:SamplerState" found in triple-slash-comments for CompareFunction defined in ../../stride/sources/engine/Stride/Graphics/CompareFunction.cs Line 14, ignored.
Invalid cref value "!:makeACopy" found in triple-slash-comments for Load defined in ../../stride/sources/engine/Stride/Graphics/Image.cs Line 506, ignored.
Invalid cref value "!:makeACopy" found in triple-slash-comments for Load defined in ../../stride/sources/engine/Stride/Graphics/Image.cs Line 506, ignored.
Invalid cref value "!:makeACopy" found in triple-slash-comments for Load defined in ../../stride/sources/engine/Stride/Graphics/Image.cs Line 520, ignored.
Invalid cref value "!:makeACopy" found in triple-slash-comments for Load defined in ../../stride/sources/engine/Stride/Graphics/Image.cs Line 520, ignored.
Invalid cref value "!:SharpDX.Direct3D11.Resource.MaximumTexture1DSize" found in triple-slash-comments for Width defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 47, ignored.
Invalid cref value "!:Texture1D" found in triple-slash-comments for Width defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 47, ignored.
Invalid cref value "!:Texture2D" found in triple-slash-comments for Width defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 47, ignored.
Invalid cref value "!:Texture3D" found in triple-slash-comments for Width defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 47, ignored.
Invalid cref value "!:TextureCube" found in triple-slash-comments for Width defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 47, ignored.
Invalid cref value "!:SharpDX.Direct3D11.Resource.MaximumTexture3DSize" found in triple-slash-comments for Height defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 55, ignored.
Invalid cref value "!:Texture2D" found in triple-slash-comments for Height defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 55, ignored.
Invalid cref value "!:Texture3D" found in triple-slash-comments for Height defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 55, ignored.
Invalid cref value "!:TextureCube" found in triple-slash-comments for Height defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 55, ignored.
Invalid cref value "!:SharpDX.Direct3D11.Resource.MaximumTexture3DSize" found in triple-slash-comments for Depth defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 63, ignored.
Invalid cref value "!:Texture3D" found in triple-slash-comments for Depth defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 63, ignored.
Invalid cref value "!:SharpDX.Direct3D11.Resource.MaximumTexture1DArraySize" found in triple-slash-comments for ArraySize defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 74, ignored.
Invalid cref value "!:Texture1D" found in triple-slash-comments for ArraySize defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 74, ignored.
Invalid cref value "!:Texture2D" found in triple-slash-comments for ArraySize defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 74, ignored.
Invalid cref value "!:TextureCube" found in triple-slash-comments for ArraySize defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 74, ignored.
Invalid cref value "!:Texture1D" found in triple-slash-comments for ArraySize defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 74, ignored.
Invalid cref value "!:Texture2D" found in triple-slash-comments for ArraySize defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 74, ignored.
Invalid cref value "!:TextureCube" found in triple-slash-comments for ArraySize defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 74, ignored.
Invalid cref value "!:SharpDX.Direct3D11.ShaderResourceViewDescription.Texture1DResource" found in triple-slash-comments for MipLevels defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 79, ignored.
Invalid cref value "!:SharpDX.DXGI.Format" found in triple-slash-comments for Format defined in ../../stride/sources/engine/Stride/Graphics/ImageDescription.cs Line 84, ignored.
Invalid cref value "!:yOffset" found in triple-slash-comments for GetPixels defined in ../../stride/sources/engine/Stride/Graphics/PixelBuffer.cs Line 228, ignored.
Invalid cref value "!:yOffset" found in triple-slash-comments for GetPixels defined in ../../stride/sources/engine/Stride/Graphics/PixelBuffer.cs Line 252, ignored.
Invalid cref value "!:pixels" found in triple-slash-comments for GetPixels defined in ../../stride/sources/engine/Stride/Graphics/PixelBuffer.cs Line 270, ignored.
Invalid cref value "!:pixels" found in triple-slash-comments for GetPixels defined in ../../stride/sources/engine/Stride/Graphics/PixelBuffer.cs Line 270, ignored.
Invalid cref value "!:yOffset" found in triple-slash-comments for GetPixels defined in ../../stride/sources/engine/Stride/Graphics/PixelBuffer.cs Line 270, ignored.
Invalid cref value "!:yOffset" found in triple-slash-comments for SetPixels defined in ../../stride/sources/engine/Stride/Graphics/PixelBuffer.cs Line 306, ignored.
Invalid cref value "!:sourcePixels" found in triple-slash-comments for SetPixels defined in ../../stride/sources/engine/Stride/Graphics/PixelBuffer.cs Line 324, ignored.
Invalid cref value "!:sourcePixels" found in triple-slash-comments for SetPixels defined in ../../stride/sources/engine/Stride/Graphics/PixelBuffer.cs Line 324, ignored.
Invalid cref value "!:yOffset" found in triple-slash-comments for SetPixels defined in ../../stride/sources/engine/Stride/Graphics/PixelBuffer.cs Line 324, ignored.
Invalid cref value "!:SamplerState" found in triple-slash-comments for Border defined in ../../stride/sources/engine/Stride/Graphics/TextureAddressMode.cs Line 30, ignored.
Processing Stride.Audio
Invalid cref value "!:ComponentBase.Dispose" found in triple-slash-comments for AudioEngine defined in ../../stride/sources/engine/Stride.Audio/AudioEngine.cs Line 18, ignored.
Invalid cref value "!:IsLooped" found in triple-slash-comments for IPlayableSound defined in ../../stride/sources/engine/Stride.Audio/IPlayableSound.cs Line 13, ignored.
Invalid cref value "!:ExitLoop" found in triple-slash-comments for IPlayableSound defined in ../../stride/sources/engine/Stride.Audio/IPlayableSound.cs Line 13, ignored.
Processing Stride.Engine
Invalid cref value "!:KeyFrameData.Time" found in triple-slash-comments for ShiftKeys defined in ../../stride/sources/engine/Stride.Engine/Animations/AnimationCurve.cs Line 71, ignored.
Invalid cref value "!:KeyFrameData.Time" found in triple-slash-comments for ShiftKeys defined in ../../stride/sources/engine/Stride.Engine/Animations/AnimationCurve.cs Line 157, ignored.
Invalid cref value "!:AudioEmitterComponent.Item(string)" found in triple-slash-comments for AudioEmitterSoundController defined in ../../stride/sources/engine/Stride.Engine/Audio/AudioEmitterSoundController.cs Line 30, ignored.
Invalid cref value "!:AudioEmitterComponent.Item(string)" found in triple-slash-comments for AudioEmitterComponent defined in ../../stride/sources/engine/Stride.Engine/Engine/AudioEmitterComponent.cs Line 29, ignored.
Invalid cref value "!:AudioEmitterComponent.Item(string)" found in triple-slash-comments for AttachSound defined in ../../stride/sources/engine/Stride.Engine/Engine/AudioEmitterComponent.cs Line 132, ignored.
D:\Projects\GitHub\stride-docs-next\en\sources\engine\Stride.Engine\Engine\Entity.cs(57,1): Duplicate parameter 'rotation' found in comments, the latter one is ignored.
Invalid cref value "!:EntityComponent.DefaultKey" found in triple-slash-comments for Add defined in ../../stride/sources/engine/Stride.Engine/Engine/Entity.cs Line 166, ignored.
Invalid cref value "!:HierarchicalProcessor" found in triple-slash-comments for Remove defined in ../../stride/sources/engine/Stride.Engine/Engine/EntityManager.cs Line 159, ignored.
Invalid cref value "!:InstancingProcessor" found in triple-slash-comments for BoundingBox defined in ../../stride/sources/engine/Stride.Engine/Engine/InstancingUserBuffer.cs Line 26, ignored.
Invalid cref value "!:RenderContext.Tags" found in triple-slash-comments for Current defined in ../../stride/sources/engine/Stride.Engine/Engine/SceneInstance.cs Line 27, ignored.
Invalid cref value "!:RenderContext.Tags" found in triple-slash-comments for CurrentRenderSystem defined in ../../stride/sources/engine/Stride.Engine/Engine/SceneInstance.cs Line 32, ignored.
Invalid cref value "!:RenderContext.Tags" found in triple-slash-comments for CurrentVisibilityGroup defined in ../../stride/sources/engine/Stride.Engine/Engine/SceneInstance.cs Line 37, ignored.
Invalid cref value "!:EffectCompilerFactory.CreateEffectCompiler" found in triple-slash-comments for EffectCompilationMode defined in ../../stride/sources/engine/Stride.Engine/Engine/Design/EffectCompilationMode.cs Line 11, ignored.
Invalid cref value "!:RouterMessage" found in triple-slash-comments for ClientRouterMessage defined in ../../stride/sources/engine/Stride.Engine/Engine/Network/ClientRouterMessage.cs Line 8, ignored.
Invalid cref value "!:Game.GraphicsDevice.Presenter.BackBuffer" found in triple-slash-comments for RenderTarget defined in ../../stride/sources/engine/Stride.Engine/Profiling/GameProfilingSystem.cs Line 67, ignored.
Invalid cref value "!:RenderContext.Tags" found in triple-slash-comments for Current defined in ../../stride/sources/engine/Stride.Engine/Rendering/CameraComponentRendererExtensions.cs Line 14, ignored.
Invalid cref value "!:RenderContext.Tags" found in triple-slash-comments for Current defined in ../../stride/sources/engine/Stride.Engine/Rendering/Compositing/GraphicsCompositor.cs Line 28, ignored.
Invalid cref value "!:RenderContext.Tags" found in triple-slash-comments for Current defined in ../../stride/sources/engine/Stride.Engine/Rendering/Compositing/SceneCameraSlotCollection.cs Line 20, ignored.
Invalid cref value "!:T" found in triple-slash-comments for ArrayUpdateResolver defined in ../../stride/sources/engine/Stride.Engine/Updater/ArrayUpdateResolver.cs Line 11, ignored.
Invalid cref value "!:target" found in triple-slash-comments for Run defined in ../../stride/sources/engine/Stride.Engine/Updater/UpdateEngine.cs Line 477, ignored.
Processing Stride.Games
Invalid cref value "!:GameForm" found in triple-slash-comments for .ctor defined in ../../stride/sources/engine/Stride.Games/SDL/GameFormSDL.cs Line 21, ignored.
Invalid cref value "!:GameForm" found in triple-slash-comments for .ctor defined in ../../stride/sources/engine/Stride.Games/SDL/GameFormSDL.cs Line 29, ignored.
Processing Stride.Graphics
Invalid cref value "!:GameContext" found in triple-slash-comments for AppContextType defined in ../../stride/sources/engine/Stride.Graphics/AppContextType.cs Line 28, ignored.
Invalid cref value "!:System.Windows.Forms.Control" found in triple-slash-comments for Desktop defined in ../../stride/sources/engine/Stride.Graphics/AppContextType.cs Line 33, ignored.
Invalid cref value "!:BlendState" found in triple-slash-comments for Blend defined in ../../stride/sources/engine/Stride.Graphics/Blend.cs Line 12, ignored.
Invalid cref value "!:GraphicsDevice.BlendStates" found in triple-slash-comments for BlendFactor defined in ../../stride/sources/engine/Stride.Graphics/Blend.cs Line 73, ignored.
Invalid cref value "!:GraphicsDevice.SetBlendState" found in triple-slash-comments for InverseBlendFactor defined in ../../stride/sources/engine/Stride.Graphics/Blend.cs Line 78, ignored.
Invalid cref value "!:BlendState" found in triple-slash-comments for BlendFunction defined in ../../stride/sources/engine/Stride.Graphics/BlendFunction.cs Line 12, ignored.
Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "T:Stride.Graphics.Buffer.Structured" -->

Invalid cref value "!:ShaderResourceView" found in triple-slash-comments for New defined in ../../stride/sources/engine/Stride.Graphics/Buffer.Structured.cs Line 45, ignored.
Invalid cref value "!:UnorderedAccessView" found in triple-slash-comments for New defined in ../../stride/sources/engine/Stride.Graphics/Buffer.Structured.cs Line 45, ignored.
Invalid cref value "!:ShaderResourceView" found in triple-slash-comments for New defined in ../../stride/sources/engine/Stride.Graphics/Buffer.Structured.cs Line 63, ignored.
Invalid cref value "!:UnorderedAccessView" found in triple-slash-comments for New defined in ../../stride/sources/engine/Stride.Graphics/Buffer.Structured.cs Line 63, ignored.
Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "T:Stride.Graphics.Buffer.StructuredAppend" -->

Invalid cref value "!:ShaderResourceView" found in triple-slash-comments for New defined in ../../stride/sources/engine/Stride.Graphics/Buffer.Structured.cs Line 125, ignored.
Invalid cref value "!:UnorderedAccessView" found in triple-slash-comments for New defined in ../../stride/sources/engine/Stride.Graphics/Buffer.Structured.cs Line 125, ignored.
Invalid cref value "!:ShaderResourceView" found in triple-slash-comments for New defined in ../../stride/sources/engine/Stride.Graphics/Buffer.Structured.cs Line 138, ignored.
Invalid cref value "!:UnorderedAccessView" found in triple-slash-comments for New defined in ../../stride/sources/engine/Stride.Graphics/Buffer.Structured.cs Line 138, ignored.
Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "T:Stride.Graphics.Buffer.StructuredCounter" -->

Invalid cref value "!:ShaderResourceView" found in triple-slash-comments for New defined in ../../stride/sources/engine/Stride.Graphics/Buffer.Structured.cs Line 186, ignored.
Invalid cref value "!:UnorderedAccessView" found in triple-slash-comments for New defined in ../../stride/sources/engine/Stride.Graphics/Buffer.Structured.cs Line 186, ignored.
Invalid cref value "!:ShaderResourceView" found in triple-slash-comments for New defined in ../../stride/sources/engine/Stride.Graphics/Buffer.Structured.cs Line 199, ignored.
Invalid cref value "!:UnorderedAccessView" found in triple-slash-comments for New defined in ../../stride/sources/engine/Stride.Graphics/Buffer.Structured.cs Line 199, ignored.
Invalid cref value "!:BindFlags.None" found in triple-slash-comments for None defined in ../../stride/sources/engine/Stride.Graphics/BufferFlags.cs Line 17, ignored.
Invalid cref value "!:BlendState" found in triple-slash-comments for ColorWriteChannels defined in ../../stride/sources/engine/Stride.Graphics/ColorWriteChannels.cs Line 13, ignored.
Invalid cref value "!:Render+states" found in triple-slash-comments for SetScissorRectangle defined in ../../stride/sources/engine/Stride.Graphics/CommandList.cs Line 148, ignored.
Invalid cref value "!:Render+states" found in triple-slash-comments for SetScissorRectangles defined in ../../stride/sources/engine/Stride.Graphics/CommandList.cs Line 160, ignored.
Invalid cref value "!:Render+states" found in triple-slash-comments for SetScissorRectangles defined in ../../stride/sources/engine/Stride.Graphics/CommandList.cs Line 170, ignored.
Invalid cref value "!:Textures+and+render+targets" found in triple-slash-comments for SetRenderTargetAndViewport defined in ../../stride/sources/engine/Stride.Graphics/CommandList.cs Line 186, ignored.
Invalid cref value "!:Textures+and+render+targets" found in triple-slash-comments for SetRenderTargetsAndViewport defined in ../../stride/sources/engine/Stride.Graphics/CommandList.cs Line 212, ignored.
Invalid cref value "!:Textures+and+render+targets" found in triple-slash-comments for SetRenderTargetsAndViewport defined in ../../stride/sources/engine/Stride.Graphics/CommandList.cs Line 224, ignored.
Invalid cref value "!:Textures+and+render+targets" found in triple-slash-comments for SetRenderTargetsAndViewport defined in ../../stride/sources/engine/Stride.Graphics/CommandList.cs Line 243, ignored.
Invalid cref value "!:Textures+and+render+targets" found in triple-slash-comments for SetRenderTarget defined in ../../stride/sources/engine/Stride.Graphics/CommandList.cs Line 268, ignored.
Invalid cref value "!:Textures+and+render+targets" found in triple-slash-comments for SetRenderTargets defined in ../../stride/sources/engine/Stride.Graphics/CommandList.cs Line 281, ignored.
Invalid cref value "!:Textures+and+render+targets" found in triple-slash-comments for SetRenderTargets defined in ../../stride/sources/engine/Stride.Graphics/CommandList.cs Line 293, ignored.
Invalid cref value "!:Textures+and+render+targets" found in triple-slash-comments for SetRenderTargets defined in ../../stride/sources/engine/Stride.Graphics/CommandList.cs Line 312, ignored.
Invalid cref value "!:Textures+and+render+targets" found in triple-slash-comments for Clear defined in ../../stride/sources/engine/Stride.Graphics/Direct3D/CommandList.Direct3D.cs Line 569, ignored.
Invalid cref value "!:Textures+and+render+targets" found in triple-slash-comments for Clear defined in ../../stride/sources/engine/Stride.Graphics/Direct3D/CommandList.Direct3D.cs Line 592, ignored.
Invalid cref value "!:TextureCube" found in triple-slash-comments for CubeMapFace defined in ../../stride/sources/engine/Stride.Graphics/CubeMapFace.cs Line 9, ignored.
Invalid cref value "!:RasterizerState" found in triple-slash-comments for CullMode defined in ../../stride/sources/engine/Stride.Graphics/CullMode.cs Line 12, ignored.
Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "F:Stride.Graphics.DeviceCreationFlags.VideoSupport" -->

Invalid cref value "!:doNotWait" found in triple-slash-comments for GetData defined in ../../stride/sources/engine/Stride.Graphics/Texture.cs Line 726, ignored.
Invalid cref value "!:doNotWait" found in triple-slash-comments for GetData defined in ../../stride/sources/engine/Stride.Graphics/Texture.cs Line 757, ignored.
Invalid cref value "!:doNotWait" found in triple-slash-comments for GetData defined in ../../stride/sources/engine/Stride.Graphics/Texture.cs Line 796, ignored.
Invalid cref value "!:Draw+a+texture" found in triple-slash-comments for DrawTexture defined in ../../stride/sources/engine/Stride.Graphics/GraphicsDeviceExtensions.cs Line 44, ignored.
Invalid cref value "!:Draw+a+texture" found in triple-slash-comments for DrawTexture defined in ../../stride/sources/engine/Stride.Graphics/GraphicsDeviceExtensions.cs Line 55, ignored.
Invalid cref value "!:Draw+a+texture" found in triple-slash-comments for DrawTexture defined in ../../stride/sources/engine/Stride.Graphics/GraphicsDeviceExtensions.cs Line 67, ignored.
Invalid cref value "!:Draw+a+texture" found in triple-slash-comments for DrawTexture defined in ../../stride/sources/engine/Stride.Graphics/GraphicsDeviceExtensions.cs Line 80, ignored.
Invalid cref value "!:SwapChain" found in triple-slash-comments for GraphicsPresenter defined in ../../stride/sources/engine/Stride.Graphics/GraphicsPresenter.cs Line 35, ignored.
Invalid cref value "!:SwapChain1" found in triple-slash-comments for GraphicsPresenter defined in ../../stride/sources/engine/Stride.Graphics/GraphicsPresenter.cs Line 35, ignored.
Invalid cref value "!:GraphicsDeviceContext.Map" found in triple-slash-comments for GraphicsResourceMap defined in ../../stride/sources/engine/Stride.Graphics/GraphicsResourceMap.cs Line 9, ignored.
Invalid cref value "!:GraphicsDeviceContext.Map" found in triple-slash-comments for MapMode defined in ../../stride/sources/engine/Stride.Graphics/MapMode.cs Line 7, ignored.
Invalid cref value "!:GraphicsDevice.MapSubresource" found in triple-slash-comments for MappedResource defined in ../../stride/sources/engine/Stride.Graphics/MappedResource.cs Line 7, ignored.
Invalid cref value "!:GraphicsDevice.DrawQuad" found in triple-slash-comments for PrimitiveQuad defined in ../../stride/sources/engine/Stride.Graphics/PrimitiveQuad.cs Line 13, ignored.
Invalid cref value "!:SpriteBatch.Begin(Stride.Graphics.SpriteSortMode,Stride.Graphics.Effect)" found in triple-slash-comments for Draw defined in ../../stride/sources/engine/Stride.Graphics/SpriteExtensions.cs Line 26, ignored.
Invalid cref value "!:SpriteBatch.End()" found in triple-slash-comments for Draw defined in ../../stride/sources/engine/Stride.Graphics/SpriteExtensions.cs Line 26, ignored.
Invalid cref value "!:SpriteBatch.Begin(Stride.Graphics.SpriteSortMode,Stride.Graphics.Effect)" found in triple-slash-comments for Draw defined in ../../stride/sources/engine/Stride.Graphics/SpriteExtensions.cs Line 46, ignored.
Invalid cref value "!:SpriteBatch.End()" found in triple-slash-comments for Draw defined in ../../stride/sources/engine/Stride.Graphics/SpriteExtensions.cs Line 46, ignored.
Invalid cref value "!:SpriteBatch.Begin(Stride.Graphics.SpriteSortMode,Stride.Graphics.Effect)" found in triple-slash-comments for Draw3D defined in ../../stride/sources/engine/Stride.Graphics/SpriteExtensions.cs Line 67, ignored.
Invalid cref value "!:SpriteBatch.End()" found in triple-slash-comments for Draw3D defined in ../../stride/sources/engine/Stride.Graphics/SpriteExtensions.cs Line 67, ignored.
Invalid cref value "!:Texture1D" found in triple-slash-comments for Width defined in ../../stride/sources/engine/Stride.Graphics/TextureDescription.cs Line 45, ignored.
Invalid cref value "!:Texture2D" found in triple-slash-comments for Width defined in ../../stride/sources/engine/Stride.Graphics/TextureDescription.cs Line 45, ignored.
Invalid cref value "!:Texture3D" found in triple-slash-comments for Width defined in ../../stride/sources/engine/Stride.Graphics/TextureDescription.cs Line 45, ignored.
Invalid cref value "!:TextureCube" found in triple-slash-comments for Width defined in ../../stride/sources/engine/Stride.Graphics/TextureDescription.cs Line 45, ignored.
Invalid cref value "!:Texture2D" found in triple-slash-comments for Height defined in ../../stride/sources/engine/Stride.Graphics/TextureDescription.cs Line 53, ignored.
Invalid cref value "!:Texture3D" found in triple-slash-comments for Height defined in ../../stride/sources/engine/Stride.Graphics/TextureDescription.cs Line 53, ignored.
Invalid cref value "!:TextureCube" found in triple-slash-comments for Height defined in ../../stride/sources/engine/Stride.Graphics/TextureDescription.cs Line 53, ignored.
Invalid cref value "!:Texture3D" found in triple-slash-comments for Depth defined in ../../stride/sources/engine/Stride.Graphics/TextureDescription.cs Line 61, ignored.
Invalid cref value "!:Texture1D" found in triple-slash-comments for ArraySize defined in ../../stride/sources/engine/Stride.Graphics/TextureDescription.cs Line 72, ignored.
Invalid cref value "!:Texture2D" found in triple-slash-comments for ArraySize defined in ../../stride/sources/engine/Stride.Graphics/TextureDescription.cs Line 72, ignored.
Invalid cref value "!:TextureCube" found in triple-slash-comments for ArraySize defined in ../../stride/sources/engine/Stride.Graphics/TextureDescription.cs Line 72, ignored.
Invalid cref value "!:Texture1D" found in triple-slash-comments for ArraySize defined in ../../stride/sources/engine/Stride.Graphics/TextureDescription.cs Line 72, ignored.
Invalid cref value "!:Texture2D" found in triple-slash-comments for ArraySize defined in ../../stride/sources/engine/Stride.Graphics/TextureDescription.cs Line 72, ignored.
Invalid cref value "!:TextureCube" found in triple-slash-comments for ArraySize defined in ../../stride/sources/engine/Stride.Graphics/TextureDescription.cs Line 72, ignored.
Invalid cref value "!:Texture2D" found in triple-slash-comments for MultisampleCount defined in ../../stride/sources/engine/Stride.Graphics/TextureDescription.cs Line 90, ignored.
Invalid cref value "!:VertexBufferLayout" found in triple-slash-comments for VertexElement defined in ../../stride/sources/engine/Stride.Graphics/VertexElement.cs Line 43, ignored.
Invalid cref value "!:Direct3D11.InputElement" found in triple-slash-comments for VertexElement defined in ../../stride/sources/engine/Stride.Graphics/VertexElement.cs Line 43, ignored.
Invalid cref value "!:Direct3D11.InputElement" found in triple-slash-comments for VertexElement defined in ../../stride/sources/engine/Stride.Graphics/VertexElement.cs Line 43, ignored.
Invalid cref value "!:VertexBufferLayout.SlotIndex" found in triple-slash-comments for VertexElement defined in ../../stride/sources/engine/Stride.Graphics/VertexElement.cs Line 43, ignored.
Invalid cref value "!:VertexBufferLayout.VertexClassification" found in triple-slash-comments for VertexElement defined in ../../stride/sources/engine/Stride.Graphics/VertexElement.cs Line 43, ignored.
Invalid cref value "!:VertexBufferLayout.instanceDataStepRate" found in triple-slash-comments for VertexElement defined in ../../stride/sources/engine/Stride.Graphics/VertexElement.cs Line 43, ignored.
Invalid cref value "!:VertexBufferLayout" found in triple-slash-comments for VertexElement defined in ../../stride/sources/engine/Stride.Graphics/VertexElement.cs Line 43, ignored.
Invalid cref value "!:Direct3D11.InputElement" found in triple-slash-comments for VertexElement defined in ../../stride/sources/engine/Stride.Graphics/VertexElement.cs Line 43, ignored.
Invalid cref value "!:Draw+vertices" found in triple-slash-comments for GeometricPrimitive defined in ../../stride/sources/engine/Stride.Graphics/GeometricPrimitives/GeometricPrimitive.Torus.cs Line 82, ignored.
Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "M:Stride.Graphics.GeometricPrimitives.GeometricPrimitive`1.#ctor(Stride.Graphics.GraphicsDevice,Stride.Graphics.GeometricMeshData{`0})" -->

Invalid cref value "!:HandleCreated" found in triple-slash-comments for Show defined in ../../stride/sources/engine/Stride.Graphics/SDL/Window.cs Line 163, ignored.
Processing Stride.Input
Invalid cref value "!:inputEvents" found in triple-slash-comments for Update defined in ../../stride/sources/engine/Stride.Input/GamePadDeviceBase.cs Line 42, ignored.
Invalid cref value "!:inputEvents" found in triple-slash-comments for Update defined in ../../stride/sources/engine/Stride.Input/GamePadFromLayout.cs Line 31, ignored.
Invalid cref value "!:inputEvents" found in triple-slash-comments for Update defined in ../../stride/sources/engine/Stride.Input/IInputDevice.cs Line 35, ignored.
Invalid cref value "!:inputEvents" found in triple-slash-comments for Update defined in ../../stride/sources/engine/Stride.Input/KeyboardDeviceBase.cs Line 41, ignored.
Invalid cref value "!:inputEvents" found in triple-slash-comments for Update defined in ../../stride/sources/engine/Stride.Input/MouseDeviceBase.cs Line 30, ignored.
Invalid cref value "!:inputEvents" found in triple-slash-comments for Update defined in ../../stride/sources/engine/Stride.Input/PointerDeviceBase.cs Line 35, ignored.
Invalid cref value "!:SurfaceSizeChanged" found in triple-slash-comments for SetSurfaceSize defined in ../../stride/sources/engine/Stride.Input/PointerDeviceState.cs Line 64, ignored.
Invalid cref value "!:OnPointer" found in triple-slash-comments for ProcessPointerEvent defined in ../../stride/sources/engine/Stride.Input/PointerDeviceState.cs Line 75, ignored.
Invalid cref value "!:CurrentPointerEvents" found in triple-slash-comments for ProcessPointerEvent defined in ../../stride/sources/engine/Stride.Input/PointerDeviceState.cs Line 75, ignored.
Invalid cref value "!:inputEvents" found in triple-slash-comments for Update defined in ../../stride/sources/engine/Stride.Input/Simulated/GamePadSimulated.cs Line 46, ignored.
Invalid cref value "!:inputEvents" found in triple-slash-comments for Update defined in ../../stride/sources/engine/Stride.Input/Simulated/MouseSimulated.cs Line 34, ignored.
Invalid cref value "!:inputEvents" found in triple-slash-comments for Update defined in ../../stride/sources/engine/Stride.Input/Simulated/PointerSimulated.cs Line 31, ignored.
Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "T:Stride.Input.VirtualButtonGroup" -->

Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "P:Stride.Input.VirtualButtonGroup.IsDisjunction" -->

Processing Stride.Navigation
Processing Stride.Particles
Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "P:Stride.Particles.ParticlePool.ParticleSize" -->

Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "M:Stride.Particles.Initializers.ParticleInitializer.Initialize(Stride.Particles.ParticlePool,System.Int32,System.Int32,System.Int32)" -->

Invalid cref value "!:ParticleSorter" found in triple-slash-comments for PrepareVertexLayout defined in ../../stride/sources/engine/Stride.Particles/Materials/ParticleMaterial.cs Line 44, ignored.
Invalid cref value "!:ParticleSorter" found in triple-slash-comments for PrepareVertexLayout defined in ../../stride/sources/engine/Stride.Particles/Materials/ParticleMaterialSimple.cs Line 79, ignored.
Invalid cref value "!:ParticleSorter" found in triple-slash-comments for BuildUVCoordinates defined in ../../stride/sources/engine/Stride.Particles/Materials/UVBuilder.cs Line 22, ignored.
Invalid cref value "!:ParticleSorter" found in triple-slash-comments for BuildUVCoordinates defined in ../../stride/sources/engine/Stride.Particles/Materials/UVBuilderFlipbook.cs Line 101, ignored.
Invalid cref value "!:ParticleSorter" found in triple-slash-comments for BuildUVCoordinates defined in ../../stride/sources/engine/Stride.Particles/Materials/UVBuilderScroll.cs Line 38, ignored.
Processing Stride.Rendering
Invalid cref value "!:MeshDrawData" found in triple-slash-comments for IsSimple defined in ../../stride/sources/engine/Stride.Rendering/Extensions/SimpleExtensions.cs Line 70, ignored.
Invalid cref value "!:VertexBufferBindingData" found in triple-slash-comments for IsSimple defined in ../../stride/sources/engine/Stride.Rendering/Extensions/SimpleExtensions.cs Line 70, ignored.
Invalid cref value "!:IndexBufferBindingData" found in triple-slash-comments for IsSimple defined in ../../stride/sources/engine/Stride.Rendering/Extensions/SimpleExtensions.cs Line 70, ignored.
Invalid cref value "!:ResourceGroupLayout.GetConstantBufferOffset" found in triple-slash-comments for ConstantBufferOffsetReference defined in ../../stride/sources/engine/Stride.Rendering/Rendering/ConstantBufferOffsetReference.cs Line 7, ignored.
Invalid cref value "!:ResourceGroupLayout.GetConstantBufferOffset" found in triple-slash-comments for EffectDescriptorSetReference defined in ../../stride/sources/engine/Stride.Rendering/Rendering/EffectDescriptorSetReference.cs Line 7, ignored.
Invalid cref value "!:ResourceGroupLayout" found in triple-slash-comments for FrameResourceGroupLayout defined in ../../stride/sources/engine/Stride.Rendering/Rendering/FrameResourceGroupLayout.cs Line 8, ignored.
Invalid cref value "!:IGraphicsRenderer.Initialize" found in triple-slash-comments for GraphicsRendererCollection defined in ../../stride/sources/engine/Stride.Rendering/Rendering/GraphicsRendererCollection.cs Line 13, ignored.
Invalid cref value "!:IGraphicsRenderer.Unload" found in triple-slash-comments for GraphicsRendererCollection defined in ../../stride/sources/engine/Stride.Rendering/Rendering/GraphicsRendererCollection.cs Line 13, ignored.
Invalid cref value "!:IGraphicsRenderer.Initialize" found in triple-slash-comments for GraphicsRendererCollectionBase defined in ../../stride/sources/engine/Stride.Rendering/Rendering/GraphicsRendererCollectionBase.cs Line 19, ignored.
Invalid cref value "!:IGraphicsRenderer.Unload" found in triple-slash-comments for GraphicsRendererCollectionBase defined in ../../stride/sources/engine/Stride.Rendering/Rendering/GraphicsRendererCollectionBase.cs Line 19, ignored.
Invalid cref value "!:EntityComponentRendererBase" found in triple-slash-comments for Enabled defined in ../../stride/sources/engine/Stride.Rendering/Rendering/GraphicsRendererCollectionBase.cs Line 36, ignored.
Invalid cref value "!:CameraRendererMode" found in triple-slash-comments for IEffectMixinProvider defined in ../../stride/sources/engine/Stride.Rendering/Rendering/IEffectMixinProvider.cs Line 11, ignored.
Invalid cref value "!:SceneEffectRenderer" found in triple-slash-comments for IImageEffectRenderer defined in ../../stride/sources/engine/Stride.Rendering/Rendering/IImageEffectRenderer.cs Line 15, ignored.
D:\Projects\GitHub\stride-docs-next\en\sources\engine\Stride.Rendering\Rendering\MeshRenderFeature.cs(91,1): Duplicate parameter 'context' found in comments, the latter one is ignored.
Invalid cref value "!:PipelinePluginManager" found in triple-slash-comments for PipelinePluginContext defined in ../../stride/sources/engine/Stride.Rendering/Rendering/PipelinePluginContext.cs Line 7, ignored.
Invalid cref value "!:SceneInstance" found in triple-slash-comments for VisibilityGroup defined in ../../stride/sources/engine/Stride.Rendering/Rendering/RenderContext.cs Line 93, ignored.
Invalid cref value "!:IGraphicsRenderer.Draw" found in triple-slash-comments for RenderDrawContext defined in ../../stride/sources/engine/Stride.Rendering/Rendering/RenderDrawContext.cs Line 13, ignored.
Invalid cref value "!:StaticEffectObjectNodeReference" found in triple-slash-comments for RenderEffect defined in ../../stride/sources/engine/Stride.Rendering/Rendering/RenderEffect.cs Line 11, ignored.
Invalid cref value "!:EntityComponentRendererBase" found in triple-slash-comments for Enabled defined in ../../stride/sources/engine/Stride.Rendering/Rendering/RendererCoreBase.cs Line 52, ignored.
Invalid cref value "!:DrawEffect.DrawCore" found in triple-slash-comments for NewScopedBuffer defined in ../../stride/sources/engine/Stride.Rendering/Rendering/RendererCoreBase.cs Line 154, ignored.
Invalid cref value "!:DrawEffect.DrawCore" found in triple-slash-comments for NewScopedTypedBuffer defined in ../../stride/sources/engine/Stride.Rendering/Rendering/RendererCoreBase.cs Line 164, ignored.
D:\Projects\GitHub\stride-docs-next\en\sources\engine\Stride.Rendering\Rendering\RootEffectRenderFeature.cs(392,1): Duplicate parameter 'context' found in comments, the latter one is ignored.
Invalid cref value "!:RenderTargetPlugin" found in triple-slash-comments for RequireDepthStencilShaderResource defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Shaders/RenderTargetKeys.cs Line 37, ignored.
D:\Projects\GitHub\stride-docs-next\en\sources\engine\Stride.Rendering\Rendering\SkinningRenderFeature.cs(47,1): Duplicate parameter 'context' found in comments, the latter one is ignored.
D:\Projects\GitHub\stride-docs-next\en\sources\engine\Stride.Rendering\Rendering\TransformRenderFeature.cs(71,1): Duplicate parameter 'context' found in comments, the latter one is ignored.
Invalid cref value "!:ResourceGroupLayout" found in triple-slash-comments for ViewResourceGroupLayout defined in ../../stride/sources/engine/Stride.Rendering/Rendering/ViewResourceGroupLayout.cs Line 8, ignored.
Invalid cref value "!:EntityComponentRendererBase" found in triple-slash-comments for Enabled defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Compositing/MSAAResolver.cs Line 79, ignored.
Invalid cref value "!:GraphicsCompositor" found in triple-slash-comments for SceneRendererBase defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Compositing/SceneRendererBase.cs Line 11, ignored.
Invalid cref value "!:ComputeEffectShader.Parameters" found in triple-slash-comments for UpdateParameters defined in ../../stride/sources/engine/Stride.Rendering/Rendering/ComputeEffect/ComputeEffectShader.cs Line 77, ignored.
Invalid cref value "!:ImageEffectShader.Parameters" found in triple-slash-comments for UpdateParameters defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Images/AntiAliasing/FXAAEffect.cs Line 87, ignored.
Invalid cref value "!:ImageEffectShader.Parameters" found in triple-slash-comments for UpdateParameters defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Images/AntiAliasing/TemporalAntiAliasEffect.cs Line 83, ignored.
Invalid cref value "!:WhitePoint" found in triple-slash-comments for Threshold defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Images/BrightFilter/BrightFilter.cs Line 46, ignored.
Invalid cref value "!:ImageEffectShader.Parameters" found in triple-slash-comments for UpdateParameters defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Images/ColorCombiner/ColorCombiner.cs Line 60, ignored.
Invalid cref value "!:DrawEffect.DrawCore" found in triple-slash-comments for NewScopedRenderTarget2D defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Images/ImageEffect.cs Line 352, ignored.
Invalid cref value "!:DrawEffect.DrawCore" found in triple-slash-comments for NewScopedRenderTarget2D defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Images/ImageEffect.cs Line 371, ignored.
Invalid cref value "!:DrawEffect.DrawCore" found in triple-slash-comments for NewScopedRenderTarget2D defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Images/ImageEffect.cs Line 390, ignored.
Invalid cref value "!:Reset" found in triple-slash-comments for SetDefaultParameters defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Images/ImageEffectShader.cs Line 91, ignored.
Invalid cref value "!:ImageEffectShader.Parameters" found in triple-slash-comments for UpdateParameters defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Images/ImageEffectShader.cs Line 110, ignored.
Invalid cref value "!:Reset" found in triple-slash-comments for SetDefaultParameters defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Images/ImageScaler/ImageScaler.cs Line 69, ignored.
Invalid cref value "!:ImageEffectShader.Parameters" found in triple-slash-comments for UpdateParameters defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Images/ImageScaler/ImageScaler.cs Line 76, ignored.
Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "P:Stride.Rendering.Images.PostProcessingEffects.Fog" -->

Invalid cref value "!:ImageEffectShader.Parameters" found in triple-slash-comments for UpdateParameters defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Images/SphericalHarmonics/SphericalHarmonicsRendererEffect.cs Line 19, ignored.
Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "P:Stride.Rendering.SubsurfaceScattering.SubsurfaceScatteringBlur.NumberOfPasses" -->

Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "P:Stride.Rendering.SubsurfaceScattering.SubsurfaceScatteringBlur.JitterKernelSize" -->

Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "P:Stride.Rendering.SubsurfaceScattering.SubsurfaceScatteringBlur.ActiveRenderMode" -->

Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "T:Stride.Rendering.SubsurfaceScattering.SubsurfaceScatteringBlur.RenderMode" -->

Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "P:Stride.Rendering.Materials.MaterialHairLightAttenuationFunctionDirectional.SelectedNormalMode" -->

Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "P:Stride.Rendering.Materials.MaterialSpecularHairModelFeature.SpecularShiftRatio" -->

D:\Projects\GitHub\stride-docs-next\en\sources\engine\Stride.Rendering\Rendering\Materials\MaterialRenderFeature.cs(107,1): Duplicate parameter 'context' found in comments, the latter one is ignored.
Invalid cref value "!:MaterialStreamAttribute" found in triple-slash-comments for MaterialStreamType defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Materials/MaterialStreamType.cs Line 8, ignored.
Invalid cref value "!:CameraComponent" found in triple-slash-comments for CurrentLights defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Lights/ForwardLightingRenderFeature.cs Line 28, ignored.
Invalid cref value "!:RenderContext.Tags" found in triple-slash-comments for CurrentLights defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Lights/ForwardLightingRenderFeature.cs Line 28, ignored.
Invalid cref value "!:LightDirectionalManualPartitionMode" found in triple-slash-comments for .ctor defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Lights/LightDirectionalShadowMap.cs Line 92, ignored.
Invalid cref value "!:SkyboxComponent" found in triple-slash-comments for LightSkybox defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Lights/LightSkybox.cs Line 13, ignored.
Invalid cref value "!:LightProcessor" found in triple-slash-comments for Position defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Lights/RenderLight.cs Line 20, ignored.
Invalid cref value "!:LightProcessor" found in triple-slash-comments for Direction defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Lights/RenderLight.cs Line 27, ignored.
Invalid cref value "!:LightProcessor" found in triple-slash-comments for BoundingBox defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Lights/RenderLight.cs Line 34, ignored.
Invalid cref value "!:LightProcessor" found in triple-slash-comments for BoundingBoxExt defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Lights/RenderLight.cs Line 39, ignored.
Invalid cref value "!:LightComponent" found in triple-slash-comments for RenderLightCollection defined in ../../stride/sources/engine/Stride.Rendering/Rendering/Lights/RenderLightCollection.cs Line 12, ignored.
Processing Stride.Shaders
Invalid cref value "!:Shader" found in triple-slash-comments for Data defined in ../../stride/sources/engine/Stride.Shaders/ShaderBytecode.cs Line 30, ignored.
Processing Stride.UI
Processing Stride.VirtualReality
Processing Stride.Core.Assets
Invalid cref value "!:ProjectBuildProfile.OutputGroupDirectories" found in triple-slash-comments for OutputGroup defined in ../../stride/sources/assets/Stride.Core.Assets/Bundle.cs Line 49, ignored.
Invalid cref value "!:Asset.Base" found in triple-slash-comments for Base defined in ../../stride/sources/assets/Stride.Core.Assets/OverrideUpgraderHint.cs Line 22, ignored.
Invalid cref value "!:Asset.BaseParts" found in triple-slash-comments for Base defined in ../../stride/sources/assets/Stride.Core.Assets/OverrideUpgraderHint.cs Line 22, ignored.
Invalid cref value "!:PackageReference" found in triple-slash-comments for PackageReferenceBase defined in ../../stride/sources/assets/Stride.Core.Assets/PackageReferenceBase.cs Line 10, ignored.
Invalid cref value "!:PackageSession.Load(string,PackageSessionResult,System.Nullable<System.Threading.CancellationToken>,bool)" found in triple-slash-comments for PackageSessionResult defined in ../../stride/sources/assets/Stride.Core.Assets/PackageSessionResult.cs Line 9, ignored.
Invalid cref value "!:IYamlProxy" found in triple-slash-comments for UnloadableObjectRemover defined in ../../stride/sources/assets/Stride.Core.Assets/UnloadableObjectRemover.cs Line 15, ignored.
Invalid cref value "!:Asset.Base" found in triple-slash-comments for AssetBaseAnalysis defined in ../../stride/sources/assets/Stride.Core.Assets/Analysis/AssetBaseAnalysis.cs Line 17, ignored.
Invalid cref value "!:relativePath" found in triple-slash-comments for GetAbsolutePath defined in ../../stride/sources/assets/Stride.Core.Assets/Compiler/AssetCompilerBase.cs Line 84, ignored.
Invalid cref value "!:relativePath" found in triple-slash-comments for GetAbsolutePath defined in ../../stride/sources/assets/Stride.Core.Assets/Compiler/AssetCompilerBase.cs Line 84, ignored.
Invalid cref value "!:LoggerResult.HasErrors" found in triple-slash-comments for BuildSteps defined in ../../stride/sources/assets/Stride.Core.Assets/Compiler/AssetCompilerResult.cs Line 30, ignored.
Invalid cref value "!:AssetPartReferenceAttribute.ReferenceType" found in triple-slash-comments for IdentifiableAssetPartReference defined in ../../stride/sources/assets/Stride.Core.Assets/Serializers/IdentifiableAssetPartReference.cs Line 13, ignored.
Invalid cref value "!:AssetVisitorBase.CurrentPath" found in triple-slash-comments for MemberPath defined in ../../stride/sources/assets/Stride.Core.Assets/Visitors/AssetMemberVisitorBase.cs Line 16, ignored.
Invalid cref value "!:AssetVisitorBase.CurrentPath" found in triple-slash-comments for VisitAssetMember defined in ../../stride/sources/assets/Stride.Core.Assets/Visitors/AssetMemberVisitorBase.cs Line 92, ignored.
Invalid triple slash comment is ignored: <!-- Badly formed XML comment ignored for member "M:Stride.Core.Yaml.CollectionWithIdsSerializerBase.TransformForSerialization(Stride.Core.Reflection.ITypeDescriptor,System.Object)" -->

Processing Stride.Physics
Creating output...


Build succeeded with warning.

    291 warning(s)
    0 error(s)
```

--- a/sources/assets/Stride.Core.Assets/Yaml/CollectionWithIdsSerializerBase.cs
+++ b/sources/assets/Stride.Core.Assets/Yaml/CollectionWithIdsSerializerBase.cs
@@ -233,7 +233,7 @@ namespace Stride.Core.Yaml
         }
 
         /// <summary>
-        /// Transforms the given collection or dictionary into a dictionary of (ids, items) or a dictionary of (ids & keys, items).
+        /// Transforms the given collection or dictionary into a dictionary of (ids, items) or a dictionary of (ids &amp; keys, items).
         /// </summary>
         /// <param name="descriptor">The type descriptor of the collection.</param>
         /// <param name="collection">The collection for which to create the mapping dictionary.</param>

--- a/sources/core/Stride.Core.Serialization/Serialization/IUrlReference.cs
+++ b/sources/core/Stride.Core.Serialization/Serialization/IUrlReference.cs
@@ -5,7 +5,7 @@ namespace Stride.Core.Serialization
     /// </summary>
     public interface IUrlReference
     {
-        // <summary>
+        /// <summary>
         /// Gets the Url of the referenced asset.
         /// </summary>
         string Url { get; }

--- a/sources/core/Stride.Core.Serialization/Serialization/UrlReferenceBase.cs
+++ b/sources/core/Stride.Core.Serialization/Serialization/UrlReferenceBase.cs
@@ -38,7 +38,7 @@ namespace Stride.Core.Serialization
             this.url = url;
         }
 
-        // <summary>
+        /// <summary>
         /// Gets the Url of the referenced asset.
         /// </summary>
         [DataMember(10)]

--- a/sources/core/Stride.Core/Collections/MultiValueSortedDictionary.cs
+++ b/sources/core/Stride.Core/Collections/MultiValueSortedDictionary.cs
@@ -258,11 +258,11 @@ namespace Stride.Core.Collections
 
         /// <summary>
         /// Copies the elements of the KoderHack.MultiValueSortedDictionary&lt;TKey,TValue&gt;
-        /// to the specified array of System.Collections.Generic.KeyValuePair<TKey,TValue>
+        /// to the specified array of <see cref="System.Collections.Generic.KeyValuePair{TKey,TValue}"/>
         /// structures, starting at the specified index.
         /// </summary>
         /// <param name="array">
-        /// The one-dimensional array of System.Collections.Generic.KeyValuePair<TKey,TValue>
+        /// The one-dimensional array of <see cref="System.Collections.Generic.KeyValuePair{TKey,TValue}"/>
         /// structures that is the destination of the elements copied from the current
         /// KoderHack.MultiValueSortedDictionary&lt;TKey,TValue&gt; The array must have
         /// zero-based indexing.
@@ -825,7 +825,7 @@ namespace Stride.Core.Collections
             /// Gets the element at the current position of the enumerator.
             /// </summary>
             /// <returns>
-            /// The element in the System.Collections.Generic.SortedDictionary<TKey,TValue>
+            /// The element in the <see cref="System.Collections.Generic.SortedDictionary{TKey, TValue}"/>
             /// at the current position of the enumerator.
             /// </returns>  
             public KeyValuePair<TKey, TValue> Current => _Current;

--- a/sources/editor/Stride.Editor/EditorGame/Game/EditorServiceGame.cs
+++ b/sources/editor/Stride.Editor/EditorGame/Game/EditorServiceGame.cs
@@ -154,7 +154,7 @@ namespace Stride.Editor.EditorGame.Game
             Physics.Simulation.DisableSimulation = true;
         }
 
-        // <inheritdoc />
+        /// <inheritdoc />
         protected override void Update(GameTime gameTime)
         {
             // Keep going only if last exception has been "resolved"

--- a/sources/engine/Stride.Graphics/Buffer.Structured.cs
+++ b/sources/engine/Stride.Graphics/Buffer.Structured.cs
@@ -31,7 +31,7 @@ namespace Stride.Graphics
         /// Structured buffer helper methods.
         /// </summary>
         /// <remarks>
-        /// Example in HLSL: StructuredBuffer&lt;float4&gt; or RWStructuredBuffer&lt;float4&gt for structured buffers supporting unordered access.
+        /// Example in HLSL: StructuredBuffer&lt;float4&gt; or RWStructuredBuffer&lt;float4&gt; for structured buffers supporting unordered access.
         /// </remarks>
         public static class Structured
         {
@@ -112,7 +112,7 @@ namespace Stride.Graphics
         /// StructuredAppend buffer helper methods.
         /// </summary>
         /// <remarks>
-        /// Example in HLSL: AppendStructuredBuffer&lt;float4&gt; or ConsumeStructuredBuffer&lt;float4&gt.
+        /// Example in HLSL: AppendStructuredBuffer&lt;float4&gt; or ConsumeStructuredBuffer&lt;float4&gt;.
         /// </remarks>
         public static class StructuredAppend
         {
@@ -173,7 +173,7 @@ namespace Stride.Graphics
         /// StructuredCounter buffer helper methods.
         /// </summary>
         /// <remarks>
-        /// Example in HLSL: StructuredBuffer&lt;float4&gt; or RWStructuredBuffer&lt;float4&gt for structured buffers supporting unordered access.
+        /// Example in HLSL: StructuredBuffer&lt;float4&gt; or RWStructuredBuffer&lt;float4&gt; for structured buffers supporting unordered access.
         /// </remarks>
         public static class StructuredCounter
         {

--- a/sources/engine/Stride.Graphics/DeviceCreationFlags.cs
+++ b/sources/engine/Stride.Graphics/DeviceCreationFlags.cs
@@ -5,7 +5,9 @@ using System;
 namespace Stride.Graphics
 {
     /// <summary>
-    /// <p>Describes parameters that are used to create a device.</p>
+    /// <para>
+    /// Describes parameters that are used to create a device.
+    /// </para>
     /// </summary>
     [Flags]
     public enum DeviceCreationFlags : int
@@ -21,7 +23,13 @@ namespace Stride.Graphics
         BgraSupport = unchecked((int)32),
 
         /// <summary>
-        /// Forces the creation of the Direct3D device to fail if the display driver is not implemented to the WDDM for Windows Developer Preview (WDDM 1.2). When the display driver is not implemented to WDDM 1.2, only a Direct3D device that is created with feature level 9.1, 9.2, or 9.3 supports video; therefore, if this flag is set, the runtime creates the Direct3D device only for feature level 9.1, 9.2, or 9.3. We recommend not to specify this flag for applications that want to favor Direct3D capability over video. If feature level 10 and higher is available, the runtime will use that feature level regardless of video support.</p> <p>If this flag is set, device creation on the Basic Render Device (BRD) will succeed regardless of the BRD's missing support for video decode. This is because the Media Foundation video stack operates in software mode on BRD. In this situation, if you force the video stack to create the Direct3D device twice (create the device once with this flag, next discover BRD, then again create the device without the flag), you actually degrade performance.</p> <p>If you attempt to create a Direct3D device with driver type <strong><see cref="SharpDX.Direct3D.DriverType.Null"/></strong>, <strong><see cref="SharpDX.Direct3D.DriverType.Reference"/></strong>, or <strong><see cref="SharpDX.Direct3D.DriverType.Software"/></strong>, device creation fails at any feature level because none of the associated drivers provide video capability. If you attempt to create a Direct3D device with driver type <strong><see cref="SharpDX.Direct3D.DriverType.Warp"/></strong>, device creation succeeds to allow software fallback for video.</p> <strong>Direct 3D 11:??</strong>This value is not supported until Direct3D 11.1. 
+        /// <para>Forces the creation of the Direct3D device to fail if the display driver is not implemented to the WDDM for Windows Developer Preview (WDDM 1.2). When the display driver is not implemented to WDDM 1.2, only a Direct3D device that is created with feature level 9.1, 9.2, or 9.3 supports video; therefore, if this flag is set, the runtime creates the Direct3D device only for feature level 9.1, 9.2, or 9.3. We recommend not to specify this flag for applications that want to favor Direct3D capability over video. If feature level 10 and higher is available, the runtime will use that feature level regardless of video support.</para>
+        /// <para>If this flag is set, device creation on the Basic Render Device (BRD) will succeed regardless of the BRD's missing support for video decode. This is because the Media Foundation video stack operates in software mode on BRD. In this situation, if you force the video stack to create the Direct3D device twice (create the device once with this flag, next discover BRD, then again create the device without the flag), you actually degrade performance.
+        /// </para>
+        /// <para>
+        /// If you attempt to create a Direct3D device with driver type <strong><see cref="SharpDX.Direct3D.DriverType.Null"/></strong>, <strong><see cref="SharpDX.Direct3D.DriverType.Reference"/></strong>, or <strong><see cref="SharpDX.Direct3D.DriverType.Software"/></strong>, device creation fails at any feature level because none of the associated drivers provide video capability. If you attempt to create a Direct3D device with driver type <strong><see cref="SharpDX.Direct3D.DriverType.Warp"/></strong>, device creation succeeds to allow software fallback for video.
+        /// </para> 
+        /// <strong>Direct 3D 11:??</strong> - This value is not supported until Direct3D 11.1. 
         /// </summary>
         VideoSupport = unchecked((int)2048),
 

--- a/sources/engine/Stride.Graphics/GeometricPrimitives/GeometricPrimitive.cs
+++ b/sources/engine/Stride.Graphics/GeometricPrimitives/GeometricPrimitive.cs
@@ -68,7 +68,7 @@ namespace Stride.Graphics.GeometricPrimitives
         /// </summary>
         /// <param name="graphicsDevice">The graphics device.</param>
         /// <param name="geometryMesh">The geometry mesh.</param>
-        /// <exception cref="System.InvalidOperationException">Cannot generate more than 65535 indices on feature level HW <= 9.3</exception>
+        /// <exception cref="System.InvalidOperationException">Cannot generate more than 65535 indices on feature level HW &lt;&#61; 9.3</exception>
         public GeometricPrimitive(GraphicsDevice graphicsDevice, GeometricMeshData<T> geometryMesh)
         {
             GraphicsDevice = graphicsDevice;

--- a/sources/engine/Stride.Input/VirtualButton/VirtualButtonGroup.cs
+++ b/sources/engine/Stride.Input/VirtualButton/VirtualButtonGroup.cs
@@ -7,7 +7,7 @@ using System.Text;
 namespace Stride.Input
 {
     /// <summary>
-    /// A combination of <see cref="IVirtualButton"/>, by default evaluated as the operator '&&' to produce a value if all buttons are pressed.
+    /// A combination <see cref="IVirtualButton"/>, by default evaluated as the operator '&amp;&amp;' to produce a value if all buttons are pressed.
     /// </summary>
     public class VirtualButtonGroup : Collection<IVirtualButton>, IVirtualButton
     {
@@ -17,7 +17,7 @@ namespace Stride.Input
         }
 
         /// <summary>
-        /// Gets or sets a value indicating whether this instance is determining the value as a disjunction ('||' operator between buttons), false by default ('&&' operator).
+        /// Gets or sets a value indicating whether this instance is determining the value as a disjunction ('&#124;&#124;' operator between buttons), false by default ('&amp;&amp;' operator).
         /// </summary>
         /// <value><c>true</c> if this instance is disjunction; otherwise, <c>false</c>.</value>
         public bool IsDisjunction { get; set; }

--- a/sources/engine/Stride.Particles/Initializers/ParticleInitializer.cs
+++ b/sources/engine/Stride.Particles/Initializers/ParticleInitializer.cs
@@ -16,7 +16,7 @@ namespace Stride.Particles.Initializers
         /// <param name="pool">Particle pool to target</param>
         /// <param name="startIdx">Starting index (included from the array)</param>
         /// <param name="endIdx">End index (excluded from the array)</param>
-        /// <param name="maxCapacity">Max pool capacity (loops after this point) so that it's possible for (endIdx < startIdx)</param>
+        /// <param name="maxCapacity">Max pool capacity (loops after this point) so that it's possible for (endIdx &lt; startIdx)</param>
         public abstract void Initialize(ParticlePool pool, int startIdx, int endIdx, int maxCapacity);
         /*
         {

--- a/sources/engine/Stride.Particles/ParticlePool.cs
+++ b/sources/engine/Stride.Particles/ParticlePool.cs
@@ -78,12 +78,9 @@ namespace Stride.Particles
         public int NextFreeIndex => nextFreeIndex;
 
         /// <summary>
-        /// Returns the size of a single particle.
-#if PARTICLES_SOA
-        /// The size of the <see cref="Particle"/> equals the sum of all fields' strides.
-#else
-        /// The size of the <see cref="Particle"/> equals the pool's stride.
-#endif
+        /// Returns the size of a single <see cref="Particle"/>.
+        /// If PARTICLES_SOA is defined, the size of the <see cref="Particle"/> is the sum of all field strides.
+        /// Otherwise, the size of the <see cref="Particle"/> is the pool's stride.
         /// </summary>
         public int ParticleSize { get; private set; }
 

--- a/sources/engine/Stride.Rendering/Rendering/Images/PostProcessingEffects.cs
+++ b/sources/engine/Stride.Rendering/Rendering/Images/PostProcessingEffects.cs
@@ -84,6 +84,7 @@ namespace Stride.Rendering.Images
         [Category]
         public Outline Outline { get; private set; }
 
+        /// <summary>
         /// Gets the fog effect.
         /// </summary>
         [DataMember(7)]

--- a/sources/engine/Stride.Rendering/Rendering/Images/SubsurfaceScattering/SubsurfaceScatteringBlur.cs
+++ b/sources/engine/Stride.Rendering/Rendering/Images/SubsurfaceScattering/SubsurfaceScatteringBlur.cs
@@ -15,28 +15,41 @@ namespace Stride.Rendering.SubsurfaceScattering
     public class SubsurfaceScatteringBlur : ImageEffect, IImageEffectRenderer, ISharedRenderer
     {
         /// <summary>
-        // Changes the render mode of the post-process for debugging purposes.
+        /// Changes the render mode of the post-process for debugging purposes.
         /// </summary>
         public enum RenderMode
         {
+            /// <summary>
+            /// Renders the scene as usual.
+            /// </summary>
             /// <userdoc>
             /// Renders the scene as usual.
             /// </userdoc>
             [Display("Default")]
             Default = 0,
 
+            /// <summary>
+            /// Shows all scattering objects in white and all other objects in black.
+            /// </summary>
             /// <userdoc>
             /// Shows all scattering objects in white and all other objects in black.
             /// </userdoc>
             [Display("Show scattering objects")]
             ShowScatteringObjects = 1,
 
+            /// <summary>
+            /// Shows the material index of scattering objects as a color.
+            /// </summary>
             /// <userdoc>
             /// Shows the material index of scattering objects as a color.
             /// </userdoc>
             [Display("Show material index")]
             ShowMaterialIndex = 2,
 
+            /// <summary>
+            /// Shows the width of the scattering kernel as a brightness value (High values will be wrapped around).
+            /// Use this to debug if each material gets its own scattering width and doesn't fluctuate.
+            /// </summary>
             /// <userdoc>
             /// Shows the width of the scattering kernel as a brightness value (High values will be wrapped around).
             /// Use this to debug if each material gets its own scattering width and doesn't fluctuate.
@@ -263,7 +276,12 @@ namespace Stride.Rendering.SubsurfaceScattering
 
             scatteringKernel.CopyTo(materialScatteringKernels, (int)materialIndex * SubsurfaceScatteringSettings.SamplesPerScatteringKernel2);   // Insert into the global scattering kernel array.
         }
-
+        
+        /// <summary>
+        /// If active, the the light won't scatter across large depth differences.
+        /// The depth falloff can be configured using "Depth falloff strength".
+        /// Attention: Enabling this increases the performance hit of the effect.
+        /// </summary>
         /// <userdoc>
         /// If active, the the light won't scatter across large depth differences.
         /// The depth falloff can be configured using "Depth falloff strength".
@@ -274,6 +292,10 @@ namespace Stride.Rendering.SubsurfaceScattering
         [Display("Follow surface")]
         public bool FollowSurface { get; set; } = true;
 
+        /// <summary>
+        /// Specifies the number of times the blur should be executed.
+        /// The higher the number of passes, the smoother the final result (less noise & banding).
+        /// </summary>
         /// <userdoc>
         /// Specifies the number of times the blur should be executed.
         /// The higher the number of passes, the smoother the final result (less noise & banding).
@@ -283,7 +305,12 @@ namespace Stride.Rendering.SubsurfaceScattering
         [DataMemberRange(1, 10, 1, 1, 0)]
         [Display("Number of passes")]
         public int NumberOfPasses { get; set; } = 1;
-
+        
+        /// <summary>
+        // This reduces the banding artifacts caused by undersampling (visible on closeups) by introducing a bit of noise.
+        // This might create a less mathematically correct falloff, since it messes with the sample offsets.
+        // But the difference is barely noticeable.
+        /// </summary>
         /// <userdoc>
         // This reduces the banding artifacts caused by undersampling (visible on closeups) by introducing a bit of noise.
         // This might create a less mathematically correct falloff, since it messes with the sample offsets.
@@ -294,6 +321,9 @@ namespace Stride.Rendering.SubsurfaceScattering
         [Display("Jitter Kernel Size")]
         public bool JitterKernelSize { get; set; } = false;
 
+        /// <summary>
+        // Changes the render mode of the post-process for debugging purposes.
+        /// </summary>
         /// <userdoc>
         // Changes the render mode of the post-process for debugging purposes.
         /// </userdoc>

--- a/sources/engine/Stride.Rendering/Rendering/Materials/Hair/LightAttenuationFunction/MaterialHairLightAttenuationFunctionDirectional.cs
+++ b/sources/engine/Stride.Rendering/Rendering/Materials/Hair/LightAttenuationFunction/MaterialHairLightAttenuationFunctionDirectional.cs
@@ -54,7 +54,7 @@ namespace Stride.Rendering.Materials
         /// Defines the types of normals to use for computing the directional attenuation.
         /// Mesh normals: Use the mesh normals.
         /// Normal map normals: Use the normal map values (falls back to mesh normals if no normal map is present).
-        /// Mesh & normal map normals: Use the normal map and mesh normals. This can fix artifacts on edges between lit and unlit areas (falls back to mesh normals if no normal map is present).
+        /// Mesh &amp; normal map normals: Use the normal map and mesh normals. This can fix artifacts on edges between lit and unlit areas (falls back to mesh normals if no normal map is present).
         /// </userdoc>
         [DataMember(20)]
         [Display("Normals to use")]

--- a/sources/engine/Stride.Rendering/Rendering/Materials/Hair/MaterialSpecularHairModelFeature.cs
+++ b/sources/engine/Stride.Rendering/Rendering/Materials/Hair/MaterialSpecularHairModelFeature.cs
@@ -131,7 +131,7 @@ namespace Stride.Rendering.Materials
         /// </summary>
         /// <userdoc>
         /// This is the ratio between the angle of the orientation of the primary and secondary specular highlights.
-        // The theoretical value is 1.5, but you can change it for more artistic control. With this parameter you can change the position of the secondary highlight.
+        /// The theoretical value is 1.5, but you can change it for more artistic control. With this parameter you can change the position of the secondary highlight.
         /// </userdoc>
         [DataMember(125)]
         [DefaultValue(1.5f)]

